api/oc_server_api.c

```diff
@@ -863,4 +874,30 @@ oc_notify_observers_delayed_ms(oc_resource_t *resource, uint16_t milliseconds)
   oc_notify_observers_delayed_ticks(resource, ticks);
 }
 
+void
+oc_notify_resource_changed(oc_resource_t *resource)
+{
+  assert(resource != NULL);
+  oc_notify_observers(resource);
```

Comment:
before invoke `oc_notify_observers` we need to check whether resource has flag observable